ui/core/src/utils/is-empty-object.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Determines if an object is an empty object
+ */
+export function isEmptyObject(obj: object) {
```

Comment:
Even though this is small, my vote would be not to rewrite utils and start using lodash here. 
https://lodash.com/docs/4.17.15#isEmpty

We can treeshake the library to only use utils that are consumed in the code.

Reply:
Yeah this makes sense, our lodash usage is a mess though, I'm planning to fix this up when I tackle Issue #894 (see error in comment [here](https://github.com/perses/perses/issues/894#issuecomment-1363612315))

Lodash also isn't a dependency in prometheus-plugin yet, so I want to make sure we set it up properly this time

Reply:
cc @juliepagano we might want to add this to our list of technical debt / deciding long term what we want to do about lodash usage, I'll add it as a discussion topic too